changelogs/fragments/352-k8s_secret-module.yml

```diff
@@ -0,0 +1,3 @@
+---
+major_changes:
+  - k8s_secret - New module for creating secrets, analogous to `kubectl create secret` (https://github.com/ansible-collections/community.kubernetes/pull/352).
```

Comment:
I think, there is no need for this file for the new module.

Reply:
ok thanks will be removed